docs/framework/react/quick-start.md

```diff
@@ -3,4 +3,53 @@ title: Quick Start
 id: quick-start
 ---
 
-# TODO
\ No newline at end of file
+The basic react app example to get started with the Tanstack react-store.
+
+```tsx
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { useStore } from "@tanstack/react-store";
+import { Store } from "@tanstack/store";
+
+export const store = new Store({
+  value1: 0,
+  value2: 0,
+});
+
+function useAppStore(selector = (state) => state) {
+  return useStore(store, selector);
+}
+
+const Display = ({ item }) => (
+  <div>
+    {item}: {useAppStore((state) => state[item])}
```

Comment:
Let's move `useAppStore` outside of the JSX - just for code taste :)

Reply:
Also, add a code comment for something like:

```
// This will only re-render when `state[item]` changes. If an unrelated store property changes, it won't re-render
```